src/Frontend/public/js/app.constants.js

```diff
@@ -4,5 +4,5 @@ window.defaultConfig = {
   version: '1.2.0',
   service_control_url: 'http://localhost:33333/api/',
   monitoring_urls: ['http://localhost:33633/'],
-  showPendingRetry: false,
+  showPendingRetry: true,
```

Comment:
is this meant to be changed as part of this PR?

Reply:
whoops